docs/conf/portland/2024/news/three-months-out.md

```diff
@@ -0,0 +1,76 @@
+---
+template: {{year}}/generic.html
+banner: _static/conf/images/headers/portland-2024-speakers.jpg
+og:image: _static/conf/images/headers/portland-2024-speakers.jpg
```

Comment:
Is this a default already and this is just an override?

Reply:
I couldn't find a good way to deafult it.

Reply:
Ah, sad times.